Annotation/polar_golden_set.json

```diff
@@ -651,13 +712,25 @@
             {
                 "predicate": [
                     {
-                        "str": "unknown",
+                        "str": "is unknown",
                         "lemma": "",
                         "POS": ""
                     }
                 ],
                 "type": "polar",
-                "clause": "",
+                "clause": "if there was anything found",

```

Comment:
This seems to me like the antecedent of the subject, and therefore not an embedded clause?

Reply:
To me just because they're the antecedent of the subject, doesn't mean they've not been embedded? I would argue that the EC is first merged, and if it's not moved to the subject it-insertion occurs, and therefore they are still embedded clauses. The only time I don't think its a true embedded clause with this subject stuff is when you cannot perform it-insertion, like "whether it will succeed depends on whether John is correct", * it depends on whether it will succeed xxx"

Reply:
Not sure about this, we should prob ask the supervisors when we meet